src/dotnet/Microsoft.DotNet.Core.Sdk.props

```diff
@@ -0,0 +1,67 @@
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <PropertyGroup>
+    <MSBuildAllProjects>$(MSBuildAllProjects);$(MSBuildThisFileFullPath)</MSBuildAllProjects>
+  </PropertyGroup>
+
+  <!-- User-facing configuration-agnostic defaults -->
+  <PropertyGroup>
+    <Configuration>Debug</Configuration>
+    <Platform>AnyCPU</Platform>
+
+    <FileAlignment>512</FileAlignment>
+    <GlobalExclude>bin\**;obj\**;</GlobalExclude>
+  </PropertyGroup>
+
+  <!-- User-facing configuration-specific defaults -->
+  <PropertyGroup>
+    <PlatformTarget>AnyCPU</PlatformTarget>
+    <ErrorReport>prompt</ErrorReport>
+    <WarningLevel>4</WarningLevel>
+    <NoWarn>$(NoWarn);1701</NoWarn>
+  </PropertyGroup>
+
+  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
+    <DebugSymbols>true</DebugSymbols>
+    <DebugType>full</DebugType>
+    <Optimize>false</Optimize>
+    <OutputPath>bin\Debug\</OutputPath>
+    <DefineConstants>DEBUG;TRACE</DefineConstants>
+  </PropertyGroup>
+  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' ">
+    <DebugType>pdbonly</DebugType>
+    <Optimize>true</Optimize>
+    <OutputPath>bin\Release\</OutputPath>
+    <DefineConstants>TRACE</DefineConstants>
+  </PropertyGroup>
+
+  <!-- Default settings for .NET Core build logic -->
+  <PropertyGroup>
+    <GenerateDependencyFile Condition=" '$(GenerateDependencyFile)' == '' ">true</GenerateDependencyFile>
+    <GenerateRuntimeConfigurationFiles Condition=" '$(GenerateRuntimeConfigurationFiles)' == '' ">true</GenerateRuntimeConfigurationFiles>
+    
+    <!-- This will turn off the base 'ResolveNuGetPackages' target since it is UWP specific, and .NET Core needs it to be split up -->
+    <!--<ResolveNuGetPackages>false</ResolveNuGetPackages>-->
+  </PropertyGroup>
+  
+  <PropertyGroup>
+    <NoStdLib>true</NoStdLib>
+    <!-- TODO: Need to figure out runtimes -->
+
+    <!-- TODO: Below are hacks eerhardt made on top of davkean's .props file -->
+    
+    <!-- Temp Hack: davkean had this, but that casues all references not to be copied to output dir: 
+    <CopyNuGetImplementations>false</CopyNuGetImplementations>
+         This will be set to false during "build", but set to true during "publish"
+    -->
+    <!--Setting PlatformTarget is needed when CopyNuGetImplementations is left on, or else you get 'Your project.json doesn't list 'win10' as a targeted runtime. You should add '"win10": { }' inside your "runtimes" section in your project.json, and then re-run NuGet restore' errors -->
+    <PlatformTarget>x64</PlatformTarget>
```

Comment:
This is just a point-in-time hack.  And this props file gets included at the beginning of the project, so you can override it anytime afterwards.

But I've added the Condition just in case.

---
In reply to: [71795608](https://github.com/dotnet/cli/pull/3911#discussion_r71795608) [](ancestors = 71795608)